packages/js/e2e-core-tests/src/specs/merchant/wp-admin-product-edit-details.test.js

```diff
@@ -20,10 +22,17 @@ let productId;
 const runProductEditDetailsTest = () => {
 	describe( 'Products > Edit Product', () => {
 		beforeAll( async () => {
-			productId = await createSimpleProduct();
 			await merchant.login();
 		} );
 
+		beforeEach( async () => {
+			productId = await createSimpleProduct();
```

Comment:
Is there a reason you changed this to a beforeEach() instead of beforeAll()?  There's only one test in the block, so it has the same effect but I'm just curious.

Reply:
Hi @lanej0, thanks for reviewing this PR!

I just changed this because we can gain a better level of independence if we create and remove data per test case rather than per `describe` block (as we would do with the `beforeAll` function).

It's true that we don't need it right now as there is only one test there, but it's likely that we'll add more tests in that part in the future. Therefore, considering the change was minimal and I was already modifying that file, I decided to follow the boy scout rule and change it there too.